### ISSUE TITLE
[Accessibility]: In the Search context dropdown, there should be separation between name and description when read by screen reader

### ISSUE THREAD

**Issue by limitedmage** [sourcegraph/sourcegraph#43134]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34188

### Problem description

The screen reader will read a search context menu item without any pause or separation between the name and the description.

<img width="677" alt="image" src="https://user-images.githubusercontent.com/206864/196556059-da5b68cc-706b-4304-b64a-ab876d9e229f.png">

### Expected behavior

We should use some kind of hidden comma to add a brief pause.

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [X] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by GitStart. I have assigned the label 'gitstart' and added the assignee 'gitstart-sourcegraph'.

/cc @lguychard
